charts/rancher-csp-adapter/0.1.0+up0.1.0-rc5/values.yaml

```diff
@@ -0,0 +1,19 @@
+debug: false
+
+image:
+  repository: rancher/rancher-csp-adapter
+  tag: v0.1.0-rc5
+  imagePullPolicy: IfNotPresent
+
+global:
+  cattle:
+    systemDefaultRegistry: ""
+  hostNetwork: false
+
+tolerations: []

```

Comment:
Tolerations are not used? Also we should have nodeSelector here too and add it to the deployment

Reply:
This would be local cluster only app. Won't support windows clusters.